### PR TITLE
Add automatic deployment

### PR DESCRIPTION
**Pull request by fspoettel**
- Release to npm on new tag via Travis integration
- Release new version of docs on push to master
- Fix asset-paths in docs-app

### CODE REVIEW

**Comment by wwwdata** [comment]
I currently don't understand how the es5 transpiling is triggered before `npm publish` which is made with the travis plugin. Did you forget to do that?

**Comment by wwwdata** [comment]
ah, and don't comment with the ci user 😄

**Comment by fspoettel** [comment]
sorry about that! 😓 you are right, the whole transpiling step should be set as prepublish-script. i am surprised that this is not the case anyways. I will add it and update the documentation accordingly.

File: src/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export * from './components/Button';
 export * from './components/ButtonGroup';
 export * from './components/Card';
 export * from './components/DataTable';
-export * from './components/Form';
+  export * from './components/Form';
```
Comment (wwwdata) [inline src/index.js]: whät?

File: package.json [hunk 2]
```diff
@@ -15,12 +14,15 @@
...
     "build": "npm run build:babel && npm run build:copy-files",
     "build:babel": "babel ./src --ignore *.spec.js --out-dir ./dist",
     "build:copy-files": "cp ./src/styles.css ./dist/; cp ./README.md ./dist/",
-    "build:docs": "webpack --config ./docs/webpack.prod.config",
+    "build:docs": "rm -rf ./docs/build; webpack --config ./docs/webpack.prod.config",
+    "deploy": "bash deploy.sh",
     "prebuild": "rm -rf ./dist",
-    "lint": "eslint src docs"
+    "prepublish": "npm run prebuild; npm run build",
+    "pretest": "npm run lint"
```
Comment (wwwdata) [inline package.json]: this is actually awesome. Force ppl to use damn linter. Nobody runs it locally I guess

File: src/index.js (hunk 1 above)
Comment (fspoettel) [inline src/index.js]: i broke it on purpose to check if pretest correctly returns the error-code for travis

**Comment by fspoettel** [comment]
I tested the gh-pages deployment locally without the travis-stuff like this: https://gist.github.com/fspoettel/59b641f3acbd24748c06ce624f54d416 and it works. Not sure if i got the travis- tuff right tho - only a push to master can tell I guess.

**Pull request merged**